src/main/java/com/intrafind/log4shell/Log4shellUpdate.java

```diff
@@ -146,6 +186,76 @@ private static void handleIfOldLog4j(Path path, Map<Path, String> toReplace, Lis
     }
   }
 
+  @SuppressWarnings("ConstantConditions")
+  private static void handleIfLog4j1(Path path, List<Path> toDelete, Map<Path, String> toAdd, Map<Path, String> toAppend, Map<Path, Map<String, String>> toReplaceInFile, Map<Path, String> toReplace) {
+    if (LOG4J1_PATTERN.asPredicate().test(path.getFileName().toString())) {
+      final Path basePath = path.getParent().getParent();
+      final Path[] slf4jImpls;
+      try {
+        slf4jImpls = Files.list(basePath.resolve("lib")).filter(file -> file.getFileName().toString().matches("slf4j-log4j12-.*\\.jar")).toArray(Path[]::new);
```

Comment:
What about using a limit to net get into stress if too much is in the path (limit(2) would be enough, right?)